stock-price-points/package.json

```diff
@@ -11,7 +11,13 @@
     "test": "jest",
     "start": "nodemon server/index.js",
     "build": "webpack --mode production",
-    "build-watch": "webpack --mode production --watch"
+    "build-watch": "webpack --mode production --watch",
+    "printId": "node data/idGenerator.js",
+    "write-data-1": "node data/dataFileGenA-C.js & node data/dataFileGenD-F.js",
```

Comment:
Are you unable to do it all at once? Just curious if its a hardware constraint. Otherwise, you can probably make a script with 
(node data/dataFileGenA-C.js & node data/dataFileGenD-F.js) && (node data/dataFileGenG-I.js & node data/dataFileGenJ-L.js) && ...
But, it might not look nice lol.